### PR TITLE
Add first version of the organization view.

### LINKED ISSUE #1933: Übersichtsseite einer Organisation abbilden
Von einem juristichen Kontakt (Organisation) soll eine Übersichtseite dargestellt werden.

Icon
Metadaten (inkl. Addressen, Telefonnummern, ...)
Zugehörigkeiten (natürliche Personen)

### PR DESCRIPTION
**Pull request by phgross**
- Add organization view closes #1933
- Link persons and organizations to the corresponding detail view

bildschirmfoto 2016-07-20 um 14 19 58

@deiferni 

Styling in [separate PR](https://github.com/4teamwork/plonetheme.teamraum/pull/447) in plonetheme.teamraum.

### CODE REVIEW

File: opengever/contact/tests/test_organization_view.py [hunk 1]
```diff
@@ -0,0 +1,94 @@
...
+
+class TestOrganizationView(FunctionalTestCase):
+
+    def setUp(self):
+        super(TestOrganizationView, self).setUp()
+        self.contactfolder = create(Builder('contactfolder')
+                                    .titled(u'Kontakte'))
+
+    @browsing
+    def test_organization_wrapper_wrap_corresponding_object(self, browser):
```
Comment (deiferni) [inline opengever/contact/tests/test_organization_view.py]: wrap_corresponding_object -> wraps_organization_model

File: opengever/contact/tests/test_organization_view.py [hunk 2]
```diff
@@ -0,0 +1,94 @@
...
+
+        browser.login().open(self.contactfolder, view=org2.wrapper_id)
+        self.assertEquals([u'4teamwork AG'], browser.css('h1').text)
+
+    @browsing
+    def test_default_view_is_organization_view(self, browser):
+        organization = create(Builder('organization').named(u'4teamwork'))
+
+        browser.login().open(organization.get_url())
+        self.assertEquals([u'4teamwork'], browser.css('h1').text)
```
Comment (deiferni) [inline opengever/contact/tests/test_organization_view.py]: imho this test does not really provide any value, i'd test against the return value of `get_url` and omit the `browser` here

File: opengever/contact/tests/test_organization_view.py [hunk 3]
```diff
@@ -100,3 +100,25 @@ def test_shows_email_addresses_prefixed_with_label(self, browser):
                           browser.css('.mail dt').text)
         self.assertEquals([u'info@example.com', u'support@example.com'],
                           browser.css('.mail dd').text)
+
+    @browsing
+    def test_list_all_persons_wich_are_related_to_the_organization(self, browser):
```
Comment (deiferni) [inline opengever/contact/tests/test_organization_view.py]: lists_all_related_persons ?

File: opengever/contact/tests/test_organization_view.py [hunk 4]
```diff
@@ -122,3 +122,18 @@ def test_list_all_persons_wich_are_related_to_the_organization(self, browser):
...
+        peter = create(Builder('person')
+                       .having(firstname=u'Peter', lastname=u'M\xfcller')
+                       .in_orgs([(org1, 'CEO')]))
+
+        browser.login().open(org1.get_url())
+
+        link = browser.css('.persons a.name').first
+        self.assertEquals(u'Peter M\xfcller', link.text)
+
+        link.click()
```
Comment (deiferni) [inline opengever/contact/tests/test_organization_view.py]: you don't have to actually open, you can just compare `href` to `get_url()`

File: opengever/contact/tests/test_person_view.py [hunk 5]
```diff
@@ -147,3 +147,18 @@ def test_list_all_of_the_users_organizations(self, browser):
...
+        peter = create(Builder('person')
+                       .having(firstname=u'Peter', lastname=u'M\xfcller')
+                       .in_orgs([(org1, 'CEO')]))
+
+        browser.login().open(peter.get_url())
+
+        link = browser.css('.organizations a.name').first
+        self.assertEquals(u'4teamwork AG', link.text)
+
+        link.click()
```
Comment (deiferni) [inline opengever/contact/tests/test_person_view.py]: same here

**Comment by deiferni** [comment]
👍

**Comment by phgross** [comment]
@deiferni updated ...

**Pull request merged**